File: verify/verify/php/check.php
<?php
$url = 'https://api.nexmo.com/verify/check/json?' . http_build_query([
        'api_key' => NEXMO_KEY,
        'api_secret' => NEXMO_SECRET,
        'request_id' => '1178e48a940a485eaca317a1237279e5',
        'code' => '6356'
    ]);

$ch = curl_init($url);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
$response = curl_exec($ch);

File: verify/search/python/check.py
import urllib

params = {
    'api_key': API_KEY,
    'api_secret': API_SECRET,
    'request_id': 'd945c3655d9043c0b0b3941e1144f258',
}

url = 'https://api.nexmo.com/verify/search/json?' + urllib.urlencode(params)

response = urllib.urlopen(url)
print response.read()